brainframe_qt/constants/oauth.py

```diff
@@ -0,0 +1,10 @@
+COGNITO_DOMAIN = "auth-aotuai-staging.auth.us-east-1.amazoncognito.com"
+"""Cognito hosted sign-in core"""
+
```

Comment:
I currently get this error with the snap build:
```
Traceback (most recent call last):
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_client.py", line 7, in <module>
    from brainframe_qt.ui.brainframe_app import BrainFrameApplication
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/ui/__init__.py", line 1, in <module>
    from .dialogs import EULADialog
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/ui/dialogs/__init__.py", line 9, in <module>
    from .license_dialog.license_dialog import LicenseDialog
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/ui/dialogs/license_dialog/license_dialog.py", line 9, in <module>
    from brainframe_qt.util.licensing import LicenseInfo, LicenseManager
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/util/licensing/__init__.py", line 2, in <module>
    from .license_manager import LicenseManager
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/util/licensing/license_manager.py", line 10, in <module>
    from brainframe_qt.util.oauth.cognito import CognitoOAuth
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/util/oauth/cognito.py", line 17, in <module>
    from brainframe_qt.util.oauth.reply_handler import PKCEReplyHandler
  File "/snap/brainframe-client/x5/usr/local/lib/python3.6/site-packages/brainframe_qt/util/oauth/reply_handler.py", line 5, in <module>
    from PyQt5.QtNetworkAuth import QOAuthHttpServerReplyHandler
ModuleNotFoundError: No module named 'PyQt5.QtNetworkAuth'
```

To reproduce:
```
python3 package/snap/build_snap.py && snap install --dangerous dist/brainframe-client_0.28.5_amd64.snap && brainframe-client
```

Reply:
Oh looks like we need to add a few dependencies to the snap environment

Reply:
This should be resolved now thanks to @velovix